server/bin/gold/test-7.1.txt

```diff
@@ -303,7 +303,7 @@ drwxrwxr-x          - tmp
 1970-01-01T00:00:42.000000 DEBUG pbench-index.pbench-index main -- start processing list of tar balls
 1970-01-01T00:00:42.000000 INFO pbench-index.pbench-index main -- Starting /var/tmp/pbench-test-server/test-7.1/pbench/archive/fs-version-001/controller/TO-INDEX/test_7.1_1970.01.01T00.00.00.tar.xz (size 1256)
 1970-01-01T00:00:42.000000 DEBUG pbench-index.pbench-index main -- open tar ball
-1970-01-01T00:00:42.000000 WARNING pbench-index.pbench-index main -- The metadata.log file is curdled in tar ball: /var/tmp/pbench-test-server/test-7.1/pbench/archive/fs-version-001/controller/test_7.1_1970.01.01T00.00.00.tar.xz - error fetching required metadata.log fields, "No section: 'run'"
+1970-01-01T00:00:42.000000 WARNING pbench-index.pbench-index main -- Bad metadata.log file encountered: /var/tmp/pbench-test-server/test-7.1/pbench/archive/fs-version-001/controller/test_7.1_1970.01.01T00.00.00.tar.xz - error fetching required metadata.log fields, "No section: 'run'"
```

Comment:
Is there a reason to change this message?

Reply:
No, Earlier messages were written in logger directly now we are using error code comments